package.json

```diff
@@ -145,6 +145,7 @@
     "cookie-parser": "^1.4.3",
     "crypto-js": "^3.1.9-1",
     "dataloader": "^1.4.0",
+    "dd-trace": "^3.4.0",
```

Comment:
There is [a bug](https://github.com/DataDog/dd-trace-js/issues/2432) in this version which causes mongo spans not to be shown as part of the main request trace. This has now been fixed so we should upgrade as soon as they put out a new version